drivers/usb/device/usb_dc_stm32.c

```diff
@@ -36,6 +37,49 @@ LOG_MODULE_REGISTER(usb_dc_stm32);
 #error "Only one interface should be enabled at a time, OTG FS or OTG HS"
 #endif
 
+/*
+ * Some STM32U5xx parts support a USB HS PHY which is closed by the HSE input clock
```

Comment:
Typo?
```suggestion
 * Some STM32U5xx parts support a USB HS PHY which is clocked by the HSE input clock
```

Reply:
Fixed, and fixed the clock input source to talk about HSE and PLL1_P as on the stm32u5 clock tree diagram.